### PR TITLE
Updated python abstractions and created examples

### PR DESCRIPTION
**Pull request by JosephDuggan7**

### CODE REVIEW

**Comment by dthain** [comment]
@JosephDuggan7 the examples here look nice, but the PR contains a number of binary objects and log files that shouldn't be checked in, please remove them.

**Review by dthain — CHANGES_REQUESTED** [review]
Please remove the unneeded files from the PR.

**Comment by JosephDuggan7** [comment]
Removed unneeded files

File: taskvine/src/examples/vine_example_pair.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import taskvine as vine
```
Comment (btovar) [inline taskvine/src/examples/vine_example_pair.py:1]: This example fails with:
```
TypeError: 'PythonTaskNoResult' object is not iterable
```

File: taskvine/src/bindings/python3/taskvine.binding.py [hunk 2]
```diff
@@ -1980,10 +1983,15 @@ def fpairs(fn, s):
...
-                t = self.wait_for_tag(str(i), 10)
+                for key, value in tasks.items():
+                    if value == i:
+                        t_id = key
+                        break
+                t = self.wait_for_task_id(t_id, 10)
 
                 if t:
-                    results[tasks[vine_task_get_id(t)]] = vine_task_get_stdout(t)
+                    print(t.output)
```
Comment (btovar) [inline taskvine/src/bindings/python3/taskvine.binding.py:1993]: debugging statement?

**Review by btovar — CHANGES_REQUESTED** [review]

**Comment by JosephDuggan7** [comment]
Removed print statement, however I am not encountering any errors with that example

**Pull request merged**